s3backgrounddelete/s3backgrounddelete/object_recovery_scheduler.py

```diff
@@ -58,6 +60,19 @@ def isObjectLeakEntryOlderThan(leakRecord, OlderInMins = 15):
         timeDeltaInMns = math.floor(timeDelta.total_seconds()/60)
         return (timeDeltaInMns >= OlderInMins)
 
+    def sigusr_handler_callback(self, signum, frame):
```

Comment:
Can you move this handler to common code, that can be reused

Reply:
@bkirunge7 I tried a few things to check how we can move this to a common code .But what I seem to not be getting is how do i access already instantiated members of ObjectRecoveryScheduler inside the signal handler callback.
 I need to access logger functions from ObjectRecoveryScheduler  class since all logs are via self.logger
Maybe make scheduler global? But that is not such a good idea i think.
And sorry the question above might be a trivial, but i am just stepping into the world of python :)

Reply:
Done in commit 28e980084adf37d96b72eb0ec6c79fe96c6728cf